src/EditorFeatures/Test2/Rename/RenameEngineResult.vb

```diff
@@ -143,6 +144,23 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Rename
             Next
         End Sub
 
+        Public Sub AssertChangedSymbolsAreAnnotated()
+            For Each location In _resolution.RelatedLocations
+                Dim documentId = location.DocumentId
+                Dim document = _resolution.NewSolution.GetDocument(documentId)
+                Dim root = document.GetSyntaxRootSynchronously(CancellationToken.None)
+
+                Dim renameAnnotatedNodes = root.GetAnnotatedNodes(RenameSymbolAnnotation.RenameSymbolKind)
+
+                ' Not all related locations will get annotated, we just need one
+                If renameAnnotatedNodes.Any() Then
+                    Return
+                End If
+            Next
+
+            Assert.True(False, "Did not find any rename annotated nodes")
```

Comment:
So, a little rant about this: Assertions throw exceptions from xunit. We have AssertEx.Fail which also will throw an exception. Both will by caught in the Try/catch surrounding this code, which makes sad pandas like me when I'm trying to investigate why a test failed and it's because in the dispose method of something later an assertion failed while in the middle of catching an exception thrown by this code. 

TLDR: AssertEx.Fail is probably more correct here? Not really sure. Either way, the message will not be what surfaces when a test fails